.gitignore

```
@@ -13,7 +13,6 @@ dist-ssr
 *.local
 
 # Editor directories and files
-.vscode/*
 !.vscode/extensions.json
```

Maybe there is something should skip inside .vsocde folder?

```suggestion
.vscode/*
!.vscode/settings.json
!.vscode/extensions.json
```